puppet/zulip/manifests/profile/postgresql.pp

```diff
@@ -49,7 +49,7 @@
     }
     if $zulip::postgresql_common::version in ['10', '11'] {
       # PostgreSQL 11 and below used a recovery.conf file for replication
-      file { "${zulip::postgresql_base::postgresql_confdir}/recovery.conf":
+      file { "${zulip::postgresql_base::postgresql_datadir}/recovery.conf":
```

Comment:
I thought `recovery.conf` did live in in `confdir` for these older Postgres versions, and that's part of what changed in postgres 12?

Reply:
Nope.  It lives in the data directory: https://pgpedia.info/r/recovery.conf.html

...which we define as:
```
# grep data_dir /etc/postgresql/14/main/*.conf
/etc/postgresql/14/main/postgresql.conf:data_directory = '/var/lib/postgresql/14/main'		# use data in another directory

Reply:
OK, thanks for confirming.